espanol/canudos.md

```diff
@@ -195,4 +195,6 @@ Luego de su viaje extraordinario se acordó de su [máquina del tiempo](maquina/
 
 [Y de repente, sintió la necesidad de que la historia llegase a su fin](fin/fin.md)
 
-[Activo su Stand ZA WARUDO](dio/dio.md)
\ No newline at end of file
+[Activo su Stand ZA WARUDO](dio/dio.md)
+
+Y despues de varios disparos decidio ir a comer, mientras su amigo lo cubria el encendia el coche para iniciar su viaje directo al [espacio](space/space.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.